src/main/java/net/minecraftforge/common/ForgeMod.java

```diff
@@ -468,9 +474,7 @@ public ForgeMod()
 
         ForgeRegistries.ITEMS.tags().addOptionalTagDefaults(Tags.Items.ENCHANTING_FUELS, Set.of(ForgeRegistries.ITEMS.getDelegateOrThrow(Items.LAPIS_LAZULI)));
 
-        // TODO: Remove when addAlias becomes proper API, as this should be done in the DR's above.
-        addAlias(ForgeRegistries.ATTRIBUTES, new ResourceLocation("forge", "reach_distance"), new ResourceLocation("forge", "block_reach"));
-        addAlias(ForgeRegistries.ATTRIBUTES, new ResourceLocation("forge", "attack_range"), new ResourceLocation("forge", "entity_reach"));
```

Comment:
Why are these being removed? It is possible that modders might be making use of them even if they didn't exist this MC version with that name as the "official" name.

Reply:
Yeah those should've been nuked in the port, but since they were missed they probably need to stay.